src/layer/heatmap/HeatMapColoredTile.js

```diff
@@ -33,13 +33,16 @@ define([
      *  color strings.
      *  @param options.extendedWidth {Number} Optional. Minimal width that needs to be retrieved for colorization.
      *  @param options.extendedHeight {Number} Optional. Minimal height that needs to be retrieved for colorization.
+     *  @param options.opacity {Number} Optional. The opacity applied to the calculated colors. The opacity is between 
+     *      0 (fully transparent) and 1 (fully opaque). The default value is 1.
      */
     var HeatMapColoredTile = function(data, options) {
         HeatMapTile.call(this, data, options);
 
         this._extendedWidth = options.extendedWidth;
         this._extendedHeight = options.extendedHeight;
         this._gradient = this.gradient(options.intensityGradient);
+        this._opacity = options.opacity * 255 || 1;
```

Comment:
If `options.opacity` equals 0, the resulting opacity will be 1 with this change. Not what you wanted I guess.